Composer/packages/extensions/adaptive-form/src/components/fields/ExpressionField/ExpressionEditor.tsx

```diff
@@ -1,20 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import React from 'react';
-import { FieldProps } from '@bfc/extension';
-import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { FieldProps, useShellApi } from '@bfc/extension';
+import { IntellisenseTextField } from '@bfc/intellisense';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import React from 'react';
+
+let intellisenseWsUrl = window.location.origin.replace(/^https/, 'wss').replace(/^http/, 'ws');
```

Comment:
Consider if the URL change should happen outside a UI component.

Reply:
Hmm. I could potentially remove url as a prop of IntellisenseTextField and move the generation of this url to a new file under packages/extensions/intellisense/src/utils. The drawback of this is that then the url is "hardcoded" into the Intellisense package, therefore communicating to 1 single server. If I wanted to create an util where the component is used then I would have to either duplicate the util per package, or create new dependency between packages in the case of 1 util. What do you think is best?